edith-step/src/ReceiptInput.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shallow, configure, mount } from 'enzyme';
+import ReceiptInput from './ReceiptInput';
+import './setupTests.js'
+
+// Submit button calls handleSubmit when clicked
+it('should call handleSumbit when Submit button is clicked', () => {
+  const handleSubmit = jest.spyOn(ReceiptInput.prototype, 'handleSubmit');
+  const component = mount(<ReceiptInput onSubmit={ handleSubmit }/>);
```

Comment:
```javascript
const component = mount(<ReceiptInput onSubmit={ handleSubmit }/>);
```
and

```javascript
component.unmount();
```
seems to be in every method. Can we use beforeEach() and afterEach() for that ?
https://jestjs.io/docs/en/setup-teardown#repeating-setup-for-many-tests

Reply:
afterEach() seems to be working well, but beforeEach doesn't work as well due to some tests mocking functions and others not